diff -uNdr nvidia-kernel.orig/nvidiapkg-x64/kernel/nv.c nvidia-kernel/nvidiapkg-x64/kernel/nv.c
--- nvidia-kernel.orig/nvidiapkg-x64/kernel/nv.c	2012-02-07 09:07:35.000000000 +0400
+++ nvidia-kernel/nvidiapkg-x64/kernel/nv.c	2012-02-23 18:07:26.802276973 +0400
@@ -17,7 +17,7 @@
 #include "rmil.h"
 
 #if defined(MODULE_LICENSE)
-MODULE_LICENSE("NVIDIA");
+MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
 MODULE_INFO(supported, "external");
diff -uNdr nvidia-kernel.orig/nvidiapkg-x86/kernel/nv.c nvidia-kernel/nvidiapkg-x86/kernel/nv.c
--- nvidia-kernel.orig/nvidiapkg-x86/kernel/nv.c	2012-02-07 08:25:43.000000000 +0400
+++ nvidia-kernel/nvidiapkg-x86/kernel/nv.c	2012-02-23 18:07:39.626858947 +0400
@@ -17,7 +17,7 @@
 #include "rmil.h"
 
 #if defined(MODULE_LICENSE)
-MODULE_LICENSE("NVIDIA");
+MODULE_LICENSE("GPL");
 #endif
 #if defined(MODULE_INFO)
 MODULE_INFO(supported, "external");
